config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml

```diff
@@ -122,6 +122,8 @@ teams:
     - jackfrancis
     - mboersma
     - shysank
+    - nojnhuh
+    - sonasingh46
```

Comment:
@CecileRobertMichon Should we also be added to `cluster-api-provider-azure-admins`?

Reply:
(pending above)
/hold

Reply:
Yep, will do.

Reply:
/hold cancel